dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/collaborator/CollaboratorHelperImpl.java

```diff
@@ -38,6 +38,7 @@
 import com.ibm.ws.runtime.metadata.ComponentMetaData;
 import com.ibm.ws.threadContext.ComponentMetaDataAccessorImpl;
 import com.ibm.ws.webcontainer.collaborator.WebAppSecurityCollaborator;
+import com.ibm.ws.webcontainer.collaborator.WebAppTransactionCollaborator;
 import com.ibm.ws.webcontainer.webapp.WebAppDispatcherContext;
```

Comment:
update the Copyright year

Reply:
@pmd1nh Done and thank you. See commit labeled "Review changes".